mesonbuild/minstall.py

```diff
@@ -219,6 +221,10 @@ def __init__(self, options, lf):
         self.lf = lf
         self.preserved_file_count = 0
 
+    def log(self, msg):
+        if not self.options.quiet:
+            print(msg)
```

Comment:
Shouldn't we use mlog.log() instead to reuse the existing logger mechanism? That gives as bonus the opportunity to use colors later.

Reply:
Yes, we could! But minstall does not use mlog, I'd classify that as a rewrite/cleanup that should happen separately :)

Reply:
indeed, doesn't have to be done here.